tests/test_orbit_phase.py

```diff
@@ -20,6 +20,8 @@ def setUpClass(cls):
         os.chdir(datadir)
         cls.pJ1855 = "B1855+09_NANOGrav_dfg+12_modified_DD.par"
         cls.mJ1855 = m.get_model(cls.pJ1855)
+        cls.pJ1855ell1 = "B1855+09_NANOGrav_12yv3.wb.gls.par"
+        cls.mJ1855ell1 = m.get_model(cls.pJ1855ell1)
```

Comment:
Is it necessary to parse this `.par` file for every test in this class? Would it not be preferable to just parse the `.par` files for the tests where they are actually used?

Reply:
Sure.  I can do that.